Formula/podsync.rb

```diff
@@ -0,0 +1,47 @@
+class Podsync < Formula
+  desc "Turn YouTube or Vimeo channels, users, or playlists into podcast feeds"
+  homepage "https://github.com/mxpv/podsync"
+  url "https://github.com/mxpv/podsync/archive/refs/tags/v2.5.0.tar.gz"
+  sha256 "45c3afea38ef45f665f456bc542ac57fecdafed884181156df6b44624b66d6b8"
+  license "MIT"
+  head "https://github.com/mxpv/podsync.git", branch: "main"
+
+  depends_on "ffmpeg"
+  depends_on "go"
```

Comment:
Is Go needed at runtime?

Reply:
Good question! The readme was ambiguous, but to me [implied yes](https://github.com/mxpv/podsync#dependencies). Logically, I'd think you wouldn't need it once you have the binary.

I just tested `podsync` after uninstalling `go` and everything worked perfectly. Moving to a build dep.